app/views/users/show.html.erb

```diff
@@ -7,14 +7,21 @@
     <%= link_to 'Connect to Github', "/auth/github", class: "btn btn-primary mb1 bg-teal" %>
   <% end %>
   </div>
-  
+
   <h3>Account Details</h3>
   <ul>
     <li> <%= current_user.first_name  %> <%= current_user.last_name %> </li>
     <li> <%= current_user.email%> </li>
   </ul>
-  <section>
+  <section class="bookmarks">
     <h1>Bookmarked Segments</h1>
+    <% @bookmarks.bookmarks.each do |bookmark| %>
```

Comment:
We may want to change the name of either the variable or the method to make this more clear later - #FIXME?